src/LinearSolvers/iterativeWrapper.jl

```diff
@@ -24,6 +24,7 @@ Example:
 
 """
 type IterativeSolver<: AbstractSolver
+   AA   # matrix being solved
```

Comment:
I'm sorry, I should have clarified that the sensitivity routines in the code I linked to in my last comment were not up to date. I updated the forward modelling but hadn't gotten to the sensitivities. I've now updated the sensitivity routines. When a linear system solve is required the code will use an existing factorization if available and not build any matrix. If factors are not being stored the code will check to see if the matrix has been stored. If so it grabs the stored matrix. Finally, if nothing has been stored the matrix is recomputed. Development of MaxwellTime has now moved to https://github.com/JuliaInv/MaxwellTime. The changes discussed in this post are reflected in the interface-refactoring branch of that repository. If you want to take a look at the code be warned that its complexity has gone up in the sensitivity routines in an effort to support inversions for conductivity and magnetic permeability in a general way.